Dockerfile

```diff
@@ -22,27 +18,27 @@ ENV LC_ALL en_US.UTF-8
 RUN rm -f /usr/bin/python \
      && ln -s /usr/bin/python3 /usr/bin/python
 
-# Create a virtual enironment to copy over into
-# the final docker image
-RUN python -m venv /opt/venv
-ENV PATH="/opt/venv/bin:$PATH"
+RUN pip3 install cirq
 
-# Copy current folder instead of cloning.
-COPY ./ .
-COPY requirements.txt .
-COPY ./cirq/contrib/contrib-requirements.txt .
-COPY ./dev_tools/conf/pip-list-dev-tools.txt .
+##cirq pre_release image
+FROM python:3.8-slim AS cirq_pre_release
 
-RUN pip3 install -r requirements.txt -r contrib-requirements.txt -r pip-list-dev-tools.txt
+# Install dependencies.
+# rm -rf /var/lib/apt/lists/* cleans up apt cache. See https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
```

Comment:
you could dedupe these to a `cirq_base` stage and use that as the base layer for the other stages `from cirq_base as cirq_stable` and `from cirq_base as cirq_pre`